CMSgov/design-system
# Wnmgds 1832 build system cleanup

## Summary

- Removes the `design-system-scripts` package from the repo
- Moves all testing output from `packages` to appropriate folders in `/tests`
- Removes `stats` scripts
- Moves all `devDependencies` into monorepo root
- Adds single `gulpfile.js` for build system
- Updated documentation to reflect changes

## How to test

1. `rm -rf node_modules && yarn install && yarn build && yarn build:healthcare && yarn build:medicare`
2. Inspect output in each packages `/dist` folder
3. `yarn test && yarn type-check`

## Checklist




### If this is a change to code:


### If this is a change to documentation:
